fastlane_core/lib/fastlane_core/configuration/configuration.rb

```diff
@@ -54,11 +54,11 @@ def verify_value_exists
       # Make sure the given value keys exist
       @values.each do |key, value|
         next if key == :trace # special treatment
-
         option = option_for_key(key)
         if option
+          @values[key] = option.auto_convert_value(value)
```

Comment:
We're already calling `auto_convert_value` in `fetch`, right? But we overwrite the `@values[key]` value. Does that mean we do it twice? Can this lead to a problem?